Sources/Soto/Extensions/DynamoDB/exports.swift

```diff
@@ -0,0 +1,15 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Soto for AWS open source project
+//
+// Copyright (c) 2024 the Soto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Soto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@_exported import _SotoDynamoDBGenerated
```

Comment:
Might need some changes for proper documentation propagation. I don't have too much experience with that so I'll defer to @0xTim who has had to deal with a bunch of exported modules in Vapor docs.

Reply:
We don't use docc on the service files. So not an issue.